aquadoggo/src/graphql/replication/client/mod.rs

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use graphql_client::GraphQLQuery;
+
+use super::Entry;
+use super::EntryHash;
+
+// The paths are relative to the directory where your `Cargo.toml` is located.
+// Both json and the GraphQL schema language are supported as sources for the schema
+#[derive(GraphQLQuery)]
+#[graphql(
+    schema_path = "src/graphql/replication/client/schema.graphql",
+    query_path = "src/graphql/replication/client/queries/get_entry_by_hash.graphql"
+)]
+pub struct GetEntryByHash;
```

Comment:
Just for understanding, I guess this is a preparation for later: We're generating a `schema.graphql` with an own `bin` from our schema definition in the code, to then re-import it here for `graphql_client`. How and where is this (`GetEntryByHash`) then used?

Reply:
> Probably this is a preparation for later

Yeah, sorry, confusing. It really belongs in the next PR.

> How and where is this (GetEntryByHash) then used?

You'll just have to wait and see ;)

Reply:
The schema is generated so the client can can do code generation. 

See https://github.com/p2panda/aquadoggo/pull/137/files#diff-cd97e251a79c2f076a0eb780a8e92a956e7c9d162350683f7a98c85ca04d8eeb for how I'm gonna use it.

Reply:
Thank you for clarifing!